home/screens/AccountModal/LoggedOutAccountView.tsx

```diff
@@ -40,7 +39,6 @@ export function LoggedOutAccountView({ refetch }: Props) {
       // after logging in, wait for redux action to dispatch, refetch with new sessionSecret, then dismiss modal
       if (isFinishedAuthenticating && sessionSecretExists) {
         try {
-          await ApolloClient.resetStore();
```

Comment:
we should keep an eye out for stale data when signing in / out in case there is some case didn't think about here